PayPalWebCheckout/src/main/java/com/paypal/android/checkoutweb/PayPalWebCheckoutFundingSource.kt

```diff
@@ -4,5 +4,18 @@ package com.paypal.android.checkoutweb
  * Enum class to specify the type of funding for an order
  */
 enum class PayPalWebCheckoutFundingSource(val value: String) {
-    CREDIT("credit"), PAY_LATER("paylater"), PAYPAL("paypal")
+    /**
+     * CREDIT will launch the web checkout flow with credit funding selected
+     */
+    CREDIT("credit"),
+
+    /**
+     * PAY_LATER will launch the web checkout flow with pay later selected
```

Comment:
```suggestion
     * PAY_LATER will launch the web checkout flow and display Pay Later offers to eligible customers after they have logged into PayPal.
```
I think these still need a little more clarity. I took this from Braintree docs, but is this accurate about what happens in the flow if they select this funding source?

Reply:
Yes, and its kinda the same for `credit`.